sources/platform/actors/development/programming_interface/metamorph.md

```diff
@@ -12,19 +12,51 @@ import TabItem from '@theme/TabItem';
 
 ---
 
-This feature is useful if you want to use another Actor to finish the work of your current one instead of internally starting a new Actor run and waiting for it to finish. With metamorph, you can easily create new Actors on top of existing ones, and give your users a nicer input structure and user interface for the final Actor. The metamorph operation is completely transparent to the Actor's users; they will just see that your Actor got the work done.
+## Transform Actor runs
 
-Internally, the system stops the Docker container corresponding to the Actor run and starts a new container using a different Docker image. All the default storages are preserved, and the new input is stored under the **INPUT-METAMORPH-1** key in the same default key-value store.
+Metamorph is a powerful operation that transforms an Actor run into the run of another Actor with a new input. This feature enables you to leverage existing Actors and create more efficient workflows.
 
-You are limited in how many times you can metamorph a single run. Check the limit in [the Actor runtime limits](/platform/limits#actor-limits).
+## Understand metamorph
 
-To make your Actor compatible with the metamorph operation, use `Actor.getInput()` instead of `Actor.getValue('INPUT')`. This method will fetch the input using the right key **INPUT-METAMORPH-1** in case of a metamorphed run.
+The metamorph process invovles several key steps. It stops the current Actor's Docker container, then starts a new container using a different Docker image. During this transition, all default storages are preserved. The new input is stored under the _INPUT-METAMORPH-1_ key in the default key-value store, ensuring seamless data transfer between Actor runs.
+
+## Benefits of metamorph
+
+Metamorph offers several advantages for developers:
```

Comment:
Advantages over what?

Reply:
Hmm, that is true, I do not provide anything to compare Metamorph with so `benefits` would make more sense, since there still is a list of positives provided for developers.